setup.py

```diff
@@ -18,6 +19,25 @@
                 "That indicates this copy of the source code is incomplete.")
         sys.exit(2)
 
+SYMLINKS = {'ansible': frozenset(('ansible-console',
```

Comment:
The glob would still be dynamic because some things in the directory are different code.